openapi-generator/src/main/kotlin/org/jellyfin/openapi/Verification.kt

```diff
@@ -0,0 +1,91 @@
+package org.jellyfin.openapi
+
+import com.squareup.kotlinpoet.FileSpec
+import mu.KotlinLogging
+import org.jellyfin.openapi.model.GeneratorResult
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.LinkOption
+import java.security.MessageDigest
+import kotlin.io.path.Path
+import kotlin.io.path.isRegularFile
+import kotlin.io.path.readText
+import kotlin.io.path.relativeTo
+import kotlin.streams.asSequence
+
+private val logger = KotlinLogging.logger { }
+
+class Verification(
+	apiOutputDir: File,
+	modelsOutputDir: File,
+) {
+	private val apiFiles by getFileTree(apiOutputDir)
+	private val modelFiles by getFileTree(modelsOutputDir)
+
+	private fun getFileTree(directory: File) = lazy {
+		assert(directory.exists() && directory.isDirectory)
+
+		val digest = MessageDigest.getInstance("MD5")
+		val path = directory.toPath()
+
+		Files
+			.walk(path)
+			.asSequence()
+			.filter { it.isRegularFile(LinkOption.NOFOLLOW_LINKS) }
+			.map {
+				val source = it
+					.readText()
+					.replace(System.lineSeparator(), "\n") // Kotlinpoet always uses \n
+					.toByteArray()
+				val hash = digest.digest(source)
+
+				it.relativeTo(path).toString() to hash
+			}
+			.toMap()
```

Comment:
Parallelizing this would be pretty cool, just saying :eyes:

Reply:
It takes like 2 seconds to run the generator/verification with our bloated openapi schema so I don't really care that much about performance :p

Reply:
*Okay, if you say so...* 😄